packages/nuxt/src/app/components/nuxt-island.ts

```diff
@@ -81,7 +82,7 @@ export default defineComponent({
         ...props.context,
         props: props.props ? JSON.stringify(props.props) : undefined
       }))
-      const result = await r.json() as NuxtIslandResponse
+      const result = process.server ? await r.json() : (r as FetchResponse<NuxtIslandResponse>)._data
```

Comment:
can we resolve this issue by just adding a `.catch` block to `r.json()`?

I also can't reproduce the issue on my end so my guess is that it's an issue with nitro's dev server socket + windows.

Reply:
Yes i already suspected that this was a windows issue. $fetch was solving this by rerunning the request. 
Should we just allow an infinite refetch on 503 instead of a single catch ? It's quite rare but i can have up to 3 failed 503 request

Reply:
Hm. It feels like this is an issue that needs to be resolved in _Nitro_ rather than worked around in this kind of way. That said, I'm happy to merge this as a hotfix if essential.

Have you tried with the latest nitropack version? There was a fix merged that should have improved things on Windows.

Reply:
you're probably referring to https://github.com/unjs/nitro/pull/1115 🤔  ? on the edge version, i still have the 503 issue (nitropack 2.5.2)

would love to have someone on windows to also confirm this

Reply:
yes, exactly. any chance you could see if you can provide a pure nitro reproduction for @pi0?

Reply:
Sure :+1: